main.cpp

```diff
@@ -176,7 +177,20 @@ int main(int argc, char **argv) {
 	wibo::Executable exec;
 	wibo::mainModule = &exec;
 
-	FILE *f = fopen(argv[1], "rb");
+	char* pe_path = argv[1];
+
+	if (!std::filesystem::exists(pe_path)) {
+		printf("File not found: %s\n", pe_path);
+		return 1;
+	}
+
```

Comment:
```suggestion
```
(covered by the below, right?)

Reply:
There are, but if the file is not found fopen will still fail and perror will append a cause like "file not found" to the error